charts/argo-workflows/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v3.4.4
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.20.12
+version: 0.20.13
```

Comment:
@danmx 
Though I approved this PR ,,,, this PR would affect some users, so maybe it's better to set `0.21.0` to let users know the impact? 🤔

Reply:
ok, makes sense. I'll change it